### PR TITLE
Browse category sidebar 

### LINKED ISSUE #122: Sidebar: Browse by Category doesn't list categories underneath
avaa-browse-category

The relevant code is [here](https://github.com/bavc/avaa/blob/master/_layouts/default.html#L64), but I'm not sure what `{% for category in categories %}` is trying to access.

### PR DESCRIPTION
**Pull request by ablwr**
Resolves #122

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -59,11 +60,11 @@ <h1 class="title"><a href="{{ site.baseurl }}/index.html"><img id="logo" src="{{
           </ul>
         </div>
         <div class="posts">
+          <h4>Browse by category</h4>
           <ul class="posts-list">
-            <h4>Browse by category</h4>
-              {% for category in categories %}
-                <li><a href="{{ site.baseurl }}/categories.html#{{ category | slugify }}"> {{ category }} </a></li>
-              {% endfor %}
+            {% for tag in tags %}
```
Comment (retokromer) [inline _layouts/default.html]: Is there no `<li>` inside the `<ul>`?

Comment (ablwr) [inline _layouts/default.html]: Ahhh thanks! I removed the `<ul>`

File: _layouts/default.html [hunk 2]
```diff
@@ -42,15 +42,16 @@ <h1 class="title"><a href="{{ site.baseurl }}/index.html"><img id="logo" src="{{
           <div class="search">
             <form action="{{ site.baseurl }}/search.html" method="get">
               <input type="text" id="search_box" name="query">
-              <input type="submit" class="search-button" value="search">
+              <input type="submit" value="search">
```
Comment (kfrn) [inline _layouts/default.html]: But why did you remove the class here? Sidebar search button is back to no styling :(

Comment (retokromer) [inline _layouts/default.html]: Sorry, I did miss that!

Comment (ablwr) [inline _layouts/default.html]: Oh, whoops! I wonder how that happened -- because I did merge master in with your changes. Will patch!
